### PR TITLE
[Merged by Bors] - chore: move some aliases earlier

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/GroupPower/Ring.lean [hunk 1]
```diff
@@ -250,3 +250,19 @@ protected theorem eq_or_eq_neg_of_sq_eq_sq (a b : Rˣ) (h : a ^ 2 = b ^ 2) : a =
...
+  rw [← pow_ne_zero_iff two_ne_zero, (sq_nonneg a).lt_iff_ne, ne_comm]
+#align sq_pos_iff sq_pos_iff
+
+alias ⟨_, sq_pos_of_ne_zero⟩ := sq_pos_iff
+#align sq_pos_of_ne_zero sq_pos_of_ne_zero
+
+alias pow_two_pos_of_ne_zero := sq_pos_of_ne_zero
+#align pow_two_pos_of_ne_zero pow_two_pos_of_ne_zero
+
+end LinearOrderedSemiring
```
[the remark this answers is not in the document]
Comment (Ruben-VandeVelde) [reply Mathlib/Algebra/GroupPower/Ring.lean]: I've reverted this.

File: Mathlib/Init/Order/Defs.lean [hunk 2]
```diff
@@ -45,6 +45,11 @@ theorem le_refl : ∀ a : α, a ≤ a :=
   Preorder.le_refl
 #align le_refl le_refl
 
+/-- A version of `le_refl` where the argument is implicit -/
+theorem le_rfl {a : α} : a ≤ a :=
+  le_refl a
+#align le_rfl le_rfl
+
```
[the remark this answers is not in the document]
Comment (Ruben-VandeVelde) [reply Mathlib/Init/Order/Defs.lean:52]: I agree with the larger point, but I do think it's important that `le_refl` and `le_rfl` are kept together, even if they'll be moved elsewhere at some point. Unless you have an open PR that conflicts with this change already, I'd like to land this.

**Review by YaelDillies — COMMENTED** [review]
Okay, let's go with this for now.

maintainer merge

**Review by Vierkantor — APPROVED** [review]
Thanks!

bors merge

**Pull request closed without merging**